app/controllers/admin/users_controller.rb

```diff
@@ -57,6 +57,10 @@ def find_user
   end
 
   def user_params
+    if params[:user][:password].blank? && params[:user][:password_confirmation].blank?
```

Comment:
đoạn này để làm gì đây c?

Reply:
đoạn này là để admin nếu chỉ muốn thay đổi role của user mà k đổi password sẽ k bị hiện tbao 
"password can't be blank"

Reply:
chỗ này c có thể thêm option `alow_blank: true` phần validate trong model `User` nhé